experimental/uefi/baremetal/src/main.rs

```diff
@@ -18,34 +18,38 @@
 #![no_main]
 #![feature(lang_items)]
 #![feature(alloc_error_handler)]
-#![feature(bench_black_box)]
+#![feature(core_ffi_c)]
 #![feature(custom_test_frameworks)]
 // As we're in a `no_std` environment, testing requires special handling. This
 // approach was inspired by https://os.phil-opp.com/testing/.
 #![test_runner(crate::test_runner)]
 #![reexport_test_harness_main = "test_main"]
 
 use core::panic::PanicInfo;
-use rust_hypervisor_firmware_boot::pvh;
+
+mod asm;
```

Comment:
asm seems to contain only multi-boot code. could the feature-gating not happen at this level?